ui/design-system-core/src/components/AppAvatar/__tests__/index.test.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import { act, cleanup } from '@testing-library/react';
+import { IntegrationTypes } from '@akashaorg/typings/ui';
+import AppAvatar from '../';
+import { customRender } from '../../../test-utils';
+
+describe('<AppAvatar /> Component', () => {
+  let componentWrapper = customRender(<></>, {});
+
+  const avatar = {
+    default: {
+      height: 320,
+      src: '',
+      width: 320,
+    },
+  };
+
+  beforeEach(() => {
+    act(() => {
+      componentWrapper = customRender(
+        <AppAvatar appType={IntegrationTypes.APP} avatar={avatar} />,
+        {},
+      );
+    });
+  });
+
+  afterEach(() => {
+    act(() => componentWrapper.unmount());
+    cleanup();
```

Comment:
Jose, I am aware this is done automatically in Jest, you can read more about this [here](https://testing-library.com/docs/react-testing-library/api/#cleanup) and [here](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-cleanup)